templates/partials/topnav.html

```diff
@@ -18,5 +21,141 @@
             </div>
         </li>
         {%- endfor -%}
+        <li class="imenu__list__item imenu__list__item--more">
+            <div class="imenu__list__anchor-container">
+                <button type="button" class="imenu__list__anchor" aria-expanded="false">
+                    Mer<span class="imenu__list__anchor-span"></span>
+                    <svg class="imenu__list__anchor-icon-right icon" focusable="false" aria-hidden>
+                        <use xlink:href="#docs-icon-arrow-down"></use>
+                    </svg>
+                </button>
+            </div>
+            <div class="ipopupmenu ipopupmenu--vertical">
+                <ul class="ipopupmenu__list ipopupmenu__list--more">
+                    <!-- prettier-ignore -->
+                    {%- for nav in topnav.children -%}
+                    {% set navPath = nav.path | replace("html", "") %}
+                    {% set navPath = navPath | replace(".", "") %}
+                    {% set activePage = navPath in doc.fileInfo.fullPath %}
+                    <li
+                        class="ipopupmenu__list__item {% if activePage %}ipopupmenu__list__item--highlight{% endif %}"
+                        data-item="nav-item-{{loop.index}}"
```

Comment:
I don't see `dataset.item` being accessed?

Reply:
- [ ] remove `data-item`, no longer used

Reply:
Fixed in ca30434821df95859e9b535e0e440e9e4ad0bbf6.